subjects/stringtointarray/README.md

```diff
@@ -0,0 +1,41 @@
+## stringtointarray
```

Comment:
It's not an array, it's a slice. The name is wrong.

Reply:
It is fixed.